core/src/main/java/org/sonarsource/sonarlint/core/websocket/SonarCloudWebSocket.java

```diff
@@ -44,7 +45,9 @@ public static String getUrl() {
     return System.getProperty("sonarlint.internal.sonarcloud.websocket.url", "wss://events-api.sonarcloud.io/");
   }
   private static final Map<String, EventParser<?>> parsersByType = Map.of(
-    "QualityGateChanged", new QualityGateChangedEventParser());
+    "QualityGateChanged", new QualityGateChangedEventParser(),
+    "MyNewIssues", new QualityGateChangedEventParser(),
```

Comment:
Bad copy/paste I think, you need to implement the right parser

Reply:
I don't think, the payload is exactly the same

Reply:
Ah ok, yes good point. Then I suggest to rename it, having a "QualityGate" parser for MyNewIssues could be confusing